### ISSUE TITLE
incoming/outgoing link popups in editor are empty

### ISSUE THREAD

**Issue by olensmar** [kubeshop/monokle#3823]
see below - I think these used to contain clickable source/target resources 

<img width="833" alt="image" src="https://user-images.githubusercontent.com/1917063/235303422-1e38e61b-c58b-4c3f-abdd-29cb7916544c.png">

<img width="816" alt="image" src="https://user-images.githubusercontent.com/1917063/235303426-e3fa1680-3718-4293-89e4-18217d1a6fc1.png">

**Comment by olensmar** [comment]
guessing this is related to #2515

**Comment by devcatalin** [comment]
I don't think we ever had any content in the glyph popups because we can only add plain text on those, no markdown or actions..
so we could remove that tooltip entirely or leave it as is, @olensmar ?

**Comment by olensmar** [comment]
hm.. this is what we had in 1.13 - not the prettiest of popups but still functional.. can we create something similar/prettier?

<img width="1689" alt="image" src="https://github.com/kubeshop/monokle/assets/1917063/5f842e82-77d6-43d0-9816-1ebb2b668c26">
